api/vehiclespecificationv1/config.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/oapi-codegen/oapi-codegen/HEAD/configuration-schema.json
+package: vehiclespecificationv1
+
+output: vehiclespecificationv1.gen.go
+
+generate:
+  models: true
+
+compatibility:
+  always-prefix-enum-values: true
+
+output-options:
+  skip-prune: true
+  name-normalizer: ToCamelCaseWithInitialisms
+  initialism-overrides: true
+  additional-initialisms:
```

Comment:
since this package is internal we can skip the naming polishing - but no need to fix since it's already there